.travis.yml

```diff
@@ -16,3 +16,4 @@ script:
   - jsonlint project.mml
   - set -o pipefail && ./node_modules/carto/bin/carto project.mml | xmllint - | wc -l
   - ./scripts/travis_check_project_files
+  - /usr/bin/xmllint --noout `find symbols/ -name \*.svg`
```

Comment:
xmllint is in the path, so no need for `/usr/bin/`. 

I'm not sure either way, but is `find | xargs xmllint` a better way to do this?